src/tikv_client.cpp

```diff
@@ -12,9 +12,68 @@ KvPair::KvPair(std::string &&key, std::string &&value)
     , value(std::move(value))
 {}
 
+ffi::KvPair KvPair::to_ffi() {
+    ffi::KvPair f_pair;
+    f_pair.key.reserve(key.size());
+    for (const auto &c : this->key) {
+        f_pair.key.emplace_back(static_cast<std::uint8_t>(c));
+    }
+    f_pair.value.reserve(value.size());
+    for (const auto &c : this->value) {
+        f_pair.value.emplace_back(static_cast<std::uint8_t>(c));
+    }
+    return f_pair;
+}
+
 TransactionClient::TransactionClient(const std::vector<std::string> &pd_endpoints):
     _client(tikv_client_glue::transaction_client_new(pd_endpoints)) {}
 
+RawKVClient::RawKVClient(const std::vector<std::string> &pd_endpoints):
+    _client(tikv_client_glue::raw_client_new(pd_endpoints)) {}
+
+std::optional<std::string> RawKVClient::get(const std::string &key, const std::uint32_t timeout) {
+    auto val = tikv_client_glue::raw_get(*_client,key,timeout);
```

Comment:
How do we handle exceptions? I noticed that `rust` lib returns a Result which may contain an error but I can't see how we can handle it.

Reply:
I think it is another topic about error handling.

Now when the [cxx](https://github.com/dtolnay/cxx) meets an rust Error, it turns the Error to be an exception in C++. However, we can not have easy way to handle different error. I've filed a issue about it https://github.com/tikv/client-cpp/issues/9

Reply:
It's another problem I want to fix because using Exceptions to handle errors in C++ is not friendly, I think it's better to error code